CloudFormation/arkcase.yml

```diff
@@ -1452,16 +1543,530 @@ Resources:
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: AlfrescoTaskPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: secretsmanager:GetSecretValue
+                Resource:
+                  - !GetAtt AlfrescoDatabaseUser.Outputs.UserSecretArn
+                  - !GetAtt Amazonmq.Outputs.AlfrescoUserSecretArn
+                Effect: Allow
+
+              - Action: route53:ChangeResourceRecordSets
+                Resource: !Sub arn:aws:route53:::hostedzone/${PrivateHostedZone}
+                Effect: Allow
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-task-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoTaskDefinition:
+    # NB: Containers in an `awsvpc` task can access each other on `localhost`
+    Type: Custom::TaskDefinition
+    Properties:
+      ServiceToken: !GetAtt TaskDefinitionResourceLambda.Arn
+      family: alfresco
+      cpu: 2048
+      memory: 8192
+      executionRoleArn: !GetAtt AlfrescoTaskExecutionRole.Arn
+      taskRoleArn: !GetAtt AlfrescoTaskRole.Arn
+      requiresCompatibilities: [ FARGATE ]
+      networkMode: awsvpc
+      volumes:
+        - name: alfresco-efs
+          efsVolumeConfiguration:
+            fileSystemId: !Ref AlfrescoEfs
+      containerDefinitions:
+        - name: acs-repository
+          image: !Sub ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/acs-repository:dev4
+          healthCheck:
+            command: [ CMD-SHELL, "curl -f http://localhost:8081/alfresco/" ]
+            interval: 10
+            retries: 2
+            startPeriod: 300  # 5 minutes; Alfresco is very slow to start but 300 seconds is the maximum allowed value
+            timeout: 3
+          # TODO XXX rm if ok to rm
+          # portMappings:
+          #   - containerPort: 8081
+          #     protocol: tcp
+          mountPoints:
+            - sourceVolume: alfresco-efs
+              containerPath: /usr/local/tomcat/alf_data
+          logConfiguration:
+            logDriver: awslogs
+            options:
+              awslogs-region: !Ref AWS::Region
+              awslogs-group: !Ref AlfrescoLogGroup
+              awslogs-stream-prefix: repository
+          essential: true
+          linuxParameters:
+            initProcessEnabled: true
+          environment:
+            - name: ARK_DB_SECRET_ARN
+              value: !GetAtt AlfrescoDatabaseUser.Outputs.UserSecretArn
+
+            - name: ARK_ACTIVEMQ_SECRET_ARN
+              value: !GetAtt Amazonmq.Outputs.AlfrescoUserSecretArn
+
+            - name: JAVA_OPTS
+              value: !Sub >
+                -Ddir.root=/usr/local/tomcat/alf_data
```

Comment:
Where does Alfresco get its LDAP configuration, so it will have the same users as Pentaho and ArkCase?